backend/Ript-Fitness-Backend/src/main/java/com/riptFitness/Ript_Fitness_Backend/infrastructure/config/JwtUtil.java

```diff
@@ -64,7 +64,7 @@ private String createToken(Map<String, Object> claims, String subject) {
                 .setClaims(claims)
                 .setSubject(subject)
                 .setIssuedAt(new Date(System.currentTimeMillis()))
-                .setExpiration(new Date(System.currentTimeMillis() + 1000 * 60 * 60 * 10)) // Token validity: 10 hours
+                .setExpiration(new Date(System.currentTimeMillis() +  1000L * 60 * 60 * 24 * 365 * 50)) // Token validity: 10 hours
```

Comment:
Please change the comment next to the .setExpiration line to indicate the change in token validity time from 10 hours to what it is now.